test/refresh.test.ts

```diff
@@ -58,25 +58,6 @@ describe("dynamic refresh", function () {
         return expect(refreshCall).eventually.rejectedWith("Refresh is not enabled for key-values or feature flags.");
     });
 
-    it("should only allow non-empty list of watched settings when refresh is enabled", async () => {

```

Comment:
Do we want the inverse test? Should refresh all when no watched settings are provided